docs/fiddles/ipc/pattern-3/preload.js

```diff
@@ -1,5 +1,6 @@
 const { contextBridge, ipcRenderer } = require('electron/renderer')
 
 contextBridge.exposeInMainWorld('electronAPI', {
-  handleCounter: (callback) => ipcRenderer.on('update-counter', () => callback())
+  onUpdateCounter: (callback) => ipcRenderer.on('update-counter', (event, value) => callback(value)),
```

Comment:
```suggestion
  onUpdateCounter: (callback) => ipcRenderer.on('update-counter', (_event, value) => callback(value)),
```

Other places in the docs we tend to use `_event` to convey that it won't be used.